pkg/cluster/cluster.go

```diff
@@ -479,6 +479,15 @@ func (c *Cluster) pollPods() ([]*v1.Pod, []*v1.Pod, error) {
 }
 
 func (c *Cluster) updateStatus() error {
+	if c.bm != nil {
+		bs, err := c.bm.getStatus()
```

Comment:
Let me give an example:
```
time="2017-02-16T18:35:53Z" level=info msg="cluster created with seed member (test-etcd-84vdc-0000)" cluster-name=test-etcd-84vdc pkg=cluster 
 
time="2017-02-16T18:36:23Z" level=error msg="failed to get backup service status: get service status (test-etcd-84vdc-backup-sidecar:19999) failed: Get http://test-etcd-84vdc-backup-sidecar:19999/v1/status: dial tcp 10.219.243.15:19999: i/o timeout"
```

Well. This time it is not DNS lookup failure. But backup pod isn't up and running yet.